src/views/News/components/NewsItem/NewsItem.module.scss

```diff
@@ -9,6 +9,10 @@
   .news_authorProfileLink {
     color: inherit;
   }
+  .news_authorProfileLink:hover {
+    text-decoration: underline;
+    text-decoration-color: #fff;
```

Comment:
At the very least, this oughta use the white variables from `_vars.scss`. Inheriting would be nice but we can at least use scss variables that can get overridden.